data/profile/src/layouts/User.vue

```diff
@@ -110,7 +110,9 @@ export default {
   },
   created () {
     const id = this.$fb.auth().currentUser.uid
```

Comment:
This isn't needed as the Vuexfire plugin handles the mutation write to the store. The `setCurrentUserData` is for when the user logs out and we set the currentUser back to null

Reply:
this is a picture from the vue dev plugin (fresh pull of the repo):
the store is not getting updated (currentUser: null), while the mutation payload contains the required data and the path does lead to "currentUser". Console log after `bindFirestoreRef` also returns `null`
![image](https://user-images.githubusercontent.com/10303647/75363229-1bb17f00-58ba-11ea-8aca-ab107b43f056.png)

Reply:
Refactor has been committed. Please update PR.